### PR TITLE
Ingest and extract obs, var, and uns metadata [VS-533] [VS-534] [VS-536] [VS-537]

### PR DESCRIPTION
**Pull request by mcovarr**
Store and extract obs and var metadata as JSON strings in cells and features respectively.

### CODE REVIEW

File: src/anndata_to_avro.py [hunk 1]
```diff
@@ -109,10 +112,9 @@ def cell_generator():
 
 
 def dump_feature_info(adata, filename, cas_feature_index_start, ingest_id):
-    # Dump out feature info -- feature_id is index (12 columns).
```
Comment (mcovarr) [inline src/anndata_to_avro.py:112]: not universally true

File: src/load_dataset.py [hunk 2]
```diff
@@ -79,6 +85,9 @@ def create_bigquery_objects(client, project, dataset):
                      bigquery.SchemaField("cas_feature_index", "INTEGER", mode="REQUIRED"),
                      bigquery.SchemaField("original_feature_id", "STRING", mode="REQUIRED"),
                      bigquery.SchemaField("feature_name", "STRING", mode="REQUIRED"),
+                     bigquery.SchemaField("var_metadata", "STRING", mode="REQUIRED"),
+                     # TODO get direct JSON metadata loading working
```
Comment (gbggrant) [inline src/load_dataset.py:89]: Is there a ticket for this?

**Review by gbggrant — APPROVED** [review]

File: src/load_dataset.py (hunk 2 above)
Comment (mcovarr) [reply src/load_dataset.py:89]: No but there should be. Kristian and I talked about it briefly in Slack on Friday afternoon and I had hoped to roll it into this PR but that didn't work out.

Comment (mcovarr) [reply src/load_dataset.py:89]: VS-546 created.

**Pull request merged**